src/ajax/ajax.js

```diff
@@ -47,45 +54,32 @@ define([ "shoestring" ], function(){
 			return;
 		}
 
-		// create parameter string from data object
-		if( settings.data ){
-			for( key in settings.data ){
-				if( settings.data.hasOwnProperty( key ) ){
-					if( params !== "" ){
-						params += "&";
-					}
-					params += encodeURIComponent( key ) + "=" +
-						encodeURIComponent( settings.data[key] );
-				}
-			}
-		}
-
-		// append params to url for GET requests
-		if( settings.method === "GET" && params ){
-			//>>includeStart("development", pragmas.development);
-			if( url.indexOf("?") >= 0 ){
-				shoestring.error( 'ajax-url-query' );
-			}
-			//>>includeEnd("development");
+		params = shoestring.ifdef({
+			call: shoestring.ajax.data.params,
```

Comment:
Should probably be changed to:

```javascript
{
  def: shoestring.ajax.data,
  call: 'params',
  ...
}
```

Because the check for definition is the parent data object.